### PR TITLE
fix: fiat input

### PR DESCRIPTION
**Pull request by AdamSchinzel**
## Description

Various fixes to fiat input in send form and exchange section.

- Show fiat input next to crypto amount only if rates are available
- Send form fiat calculation when currency select inside changed

### CODE REVIEW

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/FiatSelect/index.tsx [hunk 1]
```diff
@@ -3,20 +3,42 @@ import { Controller } from 'react-hook-form';
 import { useCoinmarketExchangeFormContext } from 'src/hooks/wallet/useCoinmarketExchangeForm';
 import { FIAT_CURRENCY } from 'src/types/wallet/coinmarketExchangeForm';
 import { buildCurrencyOptions } from '@suite-common/wallet-utils';
+import { updateFiatRatesThunk } from '@suite-common/wallet-core';
+import { Timestamp, TokenAddress } from '@suite-common/wallet-types';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { useDispatch } from 'src/hooks/suite';
+import { FiatCurrencyCode } from '@suite-common/suite-config';
 
 const FiatSelect = () => {
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/FiatSelect/index.tsx:12]: This component is never visible for Ethereum in Exchange tab. Could you please check why?

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { useCallback, useEffect } from 'react';
 import styled, { css } from 'styled-components';
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx:2]: Would be nice to see fiat rate for tokens in exchange tab
Screenshot 2024-02-27 at 17 51 52

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/FiatSelect/index.tsx (hunk 1 above)
Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/FiatSelect/index.tsx:12]: That is the inner addon for fiat input and it's visible for me.

Screenshot 2024-02-28 at 10 33 03 AM

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx (hunk 2 above)
Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx:2]: dc58eeda11835004d72cd01148c33b83c37b237c

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/FiatSelect/index.tsx (hunk 1 above)
Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/FiatInput/FiatSelect/index.tsx:12]: Should be now fixed

**Review by tomasklim — APPROVED** [review]

**Pull request merged**